tests/test_integration.py

```diff
@@ -24,7 +24,7 @@ class IntegrationTests(unittest.TestCase):
 
     def test_sudoku(self):
         demo_file = os.path.join(project_dir, 'sudoku.py')
-        output = subprocess.check_output([sys.executable, demo_file, "problem.txt"])
+        output = subprocess.check_output([sys.executable, demo_file, project_dir + "/problem.txt"])
```

Comment:
For windows compatibility, I'd suggest you use something like this:
```suggestion
        problem_file = os.path.join(project_dir, 'problem.txt')
        output = subprocess.check_output([sys.executable, demo_file, project_file])
```